doc/input-output-reference/src/overview/group-coil-system-cooling-water.tex

```diff
@@ -0,0 +1,171 @@
+
+
+\paragraph{Field: Minimum Air To Water Temperature Offset}\label{field-minimum-air-to-water-temperature-offset}
+
+The coil system will turn ON as required when coil entering air temperature is above coil entering water temperature by more than the amount of this offset. To model a waterside economizer connected to condenser loop an increased offset as desired. Default is 0.
+
+
+Following is an example input for a coil system cooling water.
+
+\begin{lstlisting}
+
+  CoilSystem:Cooling:Water,
+    Unitary-Free-Cooling,    !- Name
+    Mixed Air Node,          !- Air Inlet Node Name
+    FreeClgCoil OutletNode,  !- Air Outlet Node Name
+    ,                        !- Availability Schedule Name
+    Coil:Cooling:Water,      !- Cooling Coil Object Type
+    Free Cooling Coil,       !- Cooling Coil Name
+    CoolReheat,              !- Dehumidification Control Type
+    Yes,                     !- Run on Sensible Load
+    Yes,                     !- Run on Latent Load
+    3.0;                     !- Minimum Air To Water Temperature Offset
+	
+  CoilSystem:Cooling:Water,
+    Unitary-Free-Cooling,    !- Name
+    Mixed Air Node,          !- Air Inlet Node Name
+    FreeClgCoil OutletNode,  !- Air Outlet Node Name
+    ,                        !- Availability Schedule Name
+    Coil:Cooling:Water:DetailedGeometry, !- Cooling Coil Object Type
+    Free Cooling Coil,       !- Cooling Coil Name
+    CoolReheat,              !- Dehumidification Control Type
+    Yes,                     !- Run on Sensible Load
+    Yes,                     !- Run on Latent Load
+    3.0;                     !- Minimum Air To Water Temperature Offset
+
+  CoilSystem:Cooling:Water,
+    Unitary-Free-Cooling,    !- Name
+    Mixed Air Node,          !- Air Inlet Node Name
+    DXACFurnace InletNode,   !- Air Outlet Node Name
+    ,                        !- Availability Schedule Name
+    CoilSystem:Cooling:Water:HeatExchangerAssisted, !- Cooling Coil Object Type
+    Free Cooling Coil HXA,   !- Cooling Coil Name
+    CoolReheat,              !- Dehumidification Control Type
+    Yes,                     !- Run on Sensible Load
+    Yes,                     !- Run on Latent Load
+    3.0;                     !- Minimum Air To Water Temperature Offset
+	
+\end{lstlisting}
+
+
+\subsubsection{Outputs}\label{outputs}
+
+Following are the list of possible output variables from this coil model:
+
+\begin{itemize}
+\item HVAC, Average, Coil System Water Part Load Ratio {[]}
+\item HVAC, Average, Coil System Water Total Cooling Rate {[}W{]}
+\item HVAC, Average, Coil System Water Sensible Cooling Rate {[}W{]}
+\item HVAC, Average, Coil System Water Latent Cooling Rate {[}W{]}
+\item HVAC, Average, Coil System Water Control Status {[]}
```

Comment:
I think we'd rather have these be named "Coil System ..." (delete Water).
Some of the titles below don't match the variable names.